projects/ngx-translate-router/src/lib/localized-router.ts

```diff
@@ -5,7 +5,7 @@ import { flatten } from '@angular/compiler';
 import {ɵgetDOM as getDOM} from '@angular/platform-browser';
 import { from, of, isObservable, Observable } from 'rxjs';
 import { mergeMap, map } from 'rxjs/operators';
-import { isPromise } from '@angular/compiler/src/util';
+import { isPromise } from './util';
```

Comment:
@rafa-as & @gilsdav 
Why copying the angular method and not include the one in **rxjs**

simply switching from:
`import { isPromise } from '@angular/compiler/src/util';`
to:
`import { isPromise } from 'rxjs/internal-compatibility';`

The code maintained by rxjs:
```
/**
 * Tests to see if the object is an ES2015 (ES6) Promise
 * @see {@link https://www.ecma-international.org/ecma-262/6.0/#sec-promise-objects}
 * @param value the object to test
 */
export function isPromise(value: any): value is PromiseLike<any> {
  return !!value && typeof (<any>value).subscribe !== 'function' && typeof (value as any).then === 'function';
}
```

Reply:
An export from internal-compatibility seems to be something that shouldn't be used

Reply:
You can see how rxjs-tslint-rules discourage the usage of internal imports. This is the reason because I have copied the Angular code.

Reply:
@giacomo @gilsdav what do you think?

Reply:
@rafa-as I don't really like the `internal-compatibility` approach. Because Angular doesn't want a critical dependency and it's.
So your PR is ok